src/services/monaco.service.ts

```diff
@@ -61,15 +61,15 @@ export class MonacoService {
   }
 
   static loadLanguageConfig(asyncAPIVersion: string) {
-    const monacoInstance = window.monaco;
+    const monacoInstance = window.monaco as any;
```

Comment:
TBH I don't like it, we should extend typings for `monacoInstance.languages` with adding `yaml` property with types like `monacoInstance.languages.json`. Could you do it? You should write types in https://github.com/asyncapi/studio/blob/master/src/react-app-env.d.ts